chain/network/src/peer_manager/connected_peers.rs

```diff
@@ -29,6 +35,49 @@ pub(crate) struct ConnectedPeer {
     pub throttle_controller: ThrottleController,
     /// Encoding used for communication.
     pub encoding: Option<Encoding>,
+    pub send_accounts_data_demux: demux::Demux<Vec<Arc<SignedAccountData>>, ()>,
+}
+
+impl ConnectedPeer {
+    pub fn send_accounts_data(
+        &self,
+        data: Vec<Arc<SignedAccountData>>,
+    ) -> impl std::future::Future<Output = ()> {
+        let addr = self.addr.clone();
+        self.send_accounts_data_demux.call(
+            data,
+            |ds: Vec<Vec<Arc<SignedAccountData>>>| async move {
+                let res = ds.iter().map(|_| ()).collect();
+                let mut sum = HashMap::<_, Arc<SignedAccountData>>::new();
+                for d in ds.into_iter().flatten() {
+                    match sum.entry((d.epoch_id.clone(), d.account_id.clone())) {
+                        Entry::Occupied(mut x) => {
+                            if x.get().timestamp < d.timestamp {
+                                x.insert(d);
+                            }
+                        }
+                        Entry::Vacant(x) => {
+                            x.insert(d);
+                        }
+                    }
+                }
+                if let Err(err) = addr
+                    .send(SendMessage {
+                        message: PeerMessage::SyncAccountsData(SyncAccountsData {
+                            incremental: true,
+                            requesting_full_sync: false,
+                            accounts_data: sum.into_values().collect(),
+                        }),
+                        context: Span::current().context(),
+                    })
+                    .await
+                {
+                    error!("Failed sending incremental SyncAccountsData: {err}");
```

Comment:
```suggestion
                    tracing::error!(target: "network", ?err, "Failed sending incremental SyncAccountsData");
```


`?err` is a bit more structured (everything uses this specific name for errors). 

`"target"` is how all the code works. I wish we relied on default targets (module name), but not quite ready to fight that battle yet:)

Oh, and `tracing::` is just I think a little bit nicer, but no strong opinion.

Reply:
done, +1 for default targets